code/modules/food&drinks/drinks/drinks/drinkingglass.dm

```diff
@@ -162,6 +162,7 @@
 				icon_state = "tequilasunriseglass"
 				name = "tequila Sunrise"
 				desc = "Oh great, now you feel nostalgic about sunrises back on Terra..."
+			// Toxins Special is also in drinks.dm because shot glasses, mirror any changes there. //
```

Comment:
I'm open to suggestions. I honestly considered removing Toxins Special from glasses entirely, but left it in as a quality of life thing (to not confuse people when their drinks suddenly wouldn't mix).